src/backend/CassandraBackend.cpp

```diff
@@ -1519,6 +1613,15 @@ CassandraBackend::open(bool readOnly)
         if (!selectNFTURI_.prepareStatement(query, session_.get()))
             continue;
 
+        query.str("");
+        query << "SELECT token_id,uri FROM " << tablePrefix << "nf_token_uris"
+              << " WHERE token_id IN ? AND"
+              << " sequence <= ?"
+              << " ORDER BY sequence DESC"
+              << " PER PARTITION LIMIT 1";
```

Comment:
this query could actually replace the existing nf_token_uri select query used for `nft_info`. not sure we really need to save on memory but it's possible.

Reply:
Same as above.